src/panels/lovelace/editor/hui-dialog-edit-card.ts

```diff
@@ -88,24 +88,24 @@ export class HuiDialogEditCard extends LitElement {
                 )}</div>`
               : html`
               <hui-yaml-editor
-                .yaml="${this._configValue!.value}"
-                @yaml-changed="${this._handleYamlChanged}"
+                .yaml=${this._configValue!.value}
```

Comment:
You know the quotes are supposed to be there?

Reply:
They don't need to be. I looked at the docs and they don't use them if it's just a ${} value

Reply:
I see prettier likes them